docs/sql-ref-syntax-aux-resource-mgmt-add-jar.md

```diff
@@ -19,4 +19,31 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`ADD JAR` adds a file, JAR or archive to the list of resources. The added file can be listed using [LIST JAR](sql-ref-syntax-aux-resource-mgmt-list-jar.html).
+
+### Syntax
+{% highlight sql %}
+ADD JAR file_name
+{% endhighlight %}
+
+### Parameters
+<dl>
+ <dt><code><em>file_name</em></code></dt>
+ <dd>The name of the file, JAR or archive to be added. It could be either on a local file system or a distributed file system.</dd>
+</dl>
+
+### Examples
+{% highlight sql %}
+ add JAR /tmp/test1.jar;
+
+ add JAR /tmp/test2.zip;
+
+ ADD JAR /tmp/test3.txt;
```

Comment:
Would adding a .txt file as a JAR really work? I guess it just puts it on the classpath like a JAR. If so, that's fine. It would be an unusual usage, so I am not sure if we should include it, but if it works it's not a big deal.

Here adding a .zip file also looks a little funny, though a JAR file is just a .zip file itself. Same comment, I wonder if we should show this?